website/project/model.py

```diff
@@ -2300,6 +2303,13 @@ def update_node_wiki(self, name, content, auth):
         )
         new_page.save()
 
+        wiki_changed.send(
+            name,
```

Comment:
As mentioned in a previous comment, the first argument to `send` should be the send--in this case the node.